source/code-snippets/usage-examples/changeStream.js

```diff
@@ -0,0 +1,31 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the following with your MongoDB deployment's connection
+// string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+
+    // create a change stream on the movies collection
+    const changeStream = await collection.watch();
+    // use the '.next()' iterator to see the change events
+    const next = await changeStream.next();
+    console.log("a change to the collection happened: \t", next);
+    // process the document ...
```

Comment:
I wonder if "Add your logic to process the document here" might be more clear than "// process the document ...". The latter could be confused for an unfinished thought.

Reply:
Fixed